felix/config/config_params.go

```diff
@@ -346,6 +346,9 @@ type Config struct {
 	// Configures MTU auto-detection.
 	MTUIfacePattern *regexp.Regexp `config:"regexp;^((en|wl|ww|sl|ib)[opsx].*|(eth|wlan|wwan).*)"`
 
+	// Encapsulation information calculated from IP Pools
```

Comment:
This is calculated from more than just IP pools - it also takes into consideration the now deprecated IPIP  / VXLAN fields.